aem/aem_on_premise_install_6.5.md

```diff
@@ -0,0 +1,19 @@
+
+## Integrate with AEM 6.5.x
+
+This documentation is specific to `AEM on premise` version 6.5.x, 
+to integrate with `AEM as cloud service`, please refer to our [`AEM as cloud service` documentation](aem_skyline_install.md).
+
+To install and configure the AEM event proxy package on version 6.5.x:
+
+1. Download the latest version of the package [version 6.5.6](https://github.com/AdobeDocs/adobeio-events/files/3729022/aem-event-proxy-6.5.6.zip) 
```

Comment:
This is not the latest merged version right. We should add link to the latest version here.

Reply:
yes, will do